rllib/utils/tests/test_check_env.py

```diff
@@ -111,5 +114,14 @@ def test_step(self):
         del env
 
 
+class TestCheckBaseEnv():
```

Comment:
I'm thinking whether we should design these tests much more from the users' perspectives, as in:
1) Create a malformed config (with a bad Env class in it), 2) pass it to Trainer() or even tune.run, 3) expect the correct error.
This would make sure that we really catch the error at the highest level that the user sees.